Dockerfile

```diff
@@ -1,25 +1,26 @@
-FROM kbase/sdkpython:3.8.10
+# FROM kbase/sdkpython:3.8.10
+FROM python:3.12-bookworm
 LABEL maintainer="KBase Developers [engage@kbase.us]"
 
 WORKDIR /kb/module
-# Update
 RUN apt-get update \
     && apt-get upgrade -y && \
     update-ca-certificates && \
     apt-get -y install xvfb curl && \
+    # these are all needed by PyQt5
+    apt-get -y install libdbus-1-3 libxcb-keysyms1 libxcb-image0 libxkbcommon-x11-0 libxkbcommon0 libxcb-icccm4 libxcb-image0 libxcb-render-util0 libxcb-shape0 libxcb-xinerama0 && \
+    apt-get clean && \
```

Comment:
one of the ete3 dependencies requires a whole load of system packages that weren't installed (even as part of the standard/non-slim bookworm version).